src/Components/MultipleFields.tsx

```diff
@@ -0,0 +1,68 @@
+import { Stack, H1 } from "@deskpro/app-sdk";
+import { IAzureWorkItem, IAzureWorkItemFields } from "../types/azure/workItem";
+import { splitArrEvery2 } from "../utils/utils";
+import { workItemFields } from "../utils/workItemFields";
+import { TwoColumn } from "./TwoColumn";
+
+type Props = {
+  data: string[];
+  processName: string;
+  workItemTypeName: string;
+  item: IAzureWorkItem;
+  fieldName: string;
+};
+
+export const MultipleFields = ({
+  data,
+  processName,
+  workItemTypeName,
+  item,
+  fieldName,
+}: Props) => {
+  return (
+    <Stack vertical gap={12} style={{ width: "100%" }}>
+      <H1>{fieldName}</H1>
+      {splitArrEvery2(data).map((field, i) => {
+        const fields = workItemFields[processName][workItemTypeName];
+
+        let [leftText, rightText] = field.map(
+          (fieldName) =>
+            item.fields[
+              fields.find((nameAndField) => nameAndField.name === fieldName)
+                ?.field as keyof IAzureWorkItemFields
+            ] as string
+        );
+
+        if (
+          isNaN(Number(leftText)) &&
+          new Date(leftText).toString() !== "Invalid Date"
```

Comment:
for dates we use [date-fns](https://date-fns.org/). By example in this case use [isDate](https://date-fns.org/v2.29.3/docs/isDate)

Reply:
Same reason above, don't think it would be necessary to install a whole lib just to use one method.